qcut/docs/task/organize-project-integration.md

```
@@ -0,0 +1,504 @@
+| 5 | Add Unused Media Detection | Medium | 1 hour | None |
+| 6 | Add Project Cleanup Dialog | Medium | 1 hour | #5 |
+| 7 | Add Unit Tests | High | 1 hour | All |
+
+**Total Estimated Time:** ~6 hours
+
+---
+
+## File Summary
+
+### New Files
+| File | Purpose |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)